### ISSUE TITLE
How can 'sx' and 'sq' capture the variance of position and orientation noise?

### ISSUE THREAD

**Issue by HuynhLam** [youngguncho/PoseNet-Pytorch#4]
Hi,

As the implementation of the loss
`loss = torch.exp(-self.sx)*loss_x + self.sx + torch.exp(-self.sq)*loss_q + self.sq`
is equivalent to eq.4 in [Kendall et al. paper](http://openaccess.thecvf.com/content_cvpr_2017/papers/Kendall_Geometric_Loss_Functions_CVPR_2017_paper.pdf), I still not understand how can the trainable parameters 'sx' and 'sq' can capture the variance of position and orientation noise from the model output? I mean, how come you implement the loss this way?

Thank you very much for your time. :)

**Comment by siyuada** [comment]
Hi, I have the same question with you. 
It seems like if my `loss_x` is bigger, then the network will update `s_x` to decrease loss, which means it may decrease `exp(-sx)`,  thought in this process, `s_x` will be bigger, but it real make network give less attention to `s_x`.

I am confused about that, hope you could share your idea and discuss more!
Thanks. :)
